spec/jobs/flood_risk_engine/update_water_management_area_spec.rb

```diff
@@ -1,5 +1,17 @@
 require "rails_helper"
 
+module Test
+  module Area
+    Response = Struct.new(:area, :successful, :error) do
+      def successful?
+        successful
+      end
+    end
+
+    Area = Struct.new(:area_id, :area_name, :code, :long_name, :short_name)
+  end
+end
```

Comment:
Can we move this in a support file?

Reply:
I can do, but think I'd rather do it as a separate housekeeping exercise. I got the idea of using a struct from stuff we've done with the validators, for example https://github.com/DEFRA/defra-ruby-validators/blob/master/spec/defra_ruby/validators/business_type_validator_spec.rb

In all the validator specs, and those that are still left in [waste-exemptions-engine](https://github.com/DEFRA/waste-exemptions-engine) the struct is declared at the top of the spec.

So if we'd rather these live in `spec/support`, I'd push for cleaning them all up at once, and recording the decision. In which case I can raise an issue in [ruby-services-team](https://github.com/DEFRA/ruby-services-team) to log and track the task??

Reply:
Sounds good :)

Reply:
Created https://github.com/DEFRA/ruby-services-team/issues/65